docs/gke/iap.md

```diff
@@ -72,18 +47,23 @@ Create an OAuth Client ID to be used to identify IAP when requesting acces to us
 If you haven't already, follow the instructions in the [user_guide](https://github.com/kubeflow/kubeflow/blob/master/user_guide.md#deploy-kubeflow)
 to create a ksonnet app to deploy Kubeflow.
 
+[cert-manager](https://github.com/jetstack/cert-manager) is used to automatically request valid SSL certifiactes using the [ACME](https://en.wikipedia.org/wiki/Automated_Certificate_Management_Environment) issuer.
+
 The instructions below reference the following environment variables which you will need to set for your deployment
 
   * **CORE_NAME** The name assigned to the core Kubeflow ksonnet components (this is the name chosen when you ran `ks generate...`).
   * **ENVIRONMENT** The name of the ksonnet environment where you want to deploy Kubeflow.
   * **FQDN** The fully qualified domain name to use for your Kubeflow deployment.
   * **IP_NAME** The name of the GCP static IP that you created above and will be associated with **DOMAIN**.
   * **NAMESPACE** The namespace where Kubeflow is deployed.
-  * **SECRET_NAME** The name of the K8s secret that stores the SSL certificate.
+  * **ACCOUNT** The email address for the ACME registered account, typically the value of `gcloud config get-value account`.
```

Comment:
Can you add a comment explaining that this email will be used for certificate expiration notifications.